spec/DSPX-3456.md

```
@@ -0,0 +1,46 @@
+title: KAS: emit INFO log of configured mechanisms after startup
+status: draft
+authors: [dmihalcik@virtru.com]
+branches: [opentdf/platform:DSPX-3456-log-supported-kao-mechanisms-at-info-during-startu]
+prs: []
+created: 2026-06-03
+updated: 2026-06-03
+---
+
+# KAS: emit INFO log of configured mechanisms after startup
+
+## Summary
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)